README.md

```diff
@@ -16,7 +16,7 @@ a direct visit from the browser.
 
 ```javascript
 // Heroku defines the environment variable PORT, and requires the binding address to be 0.0.0.0
-var host = process.env.PORT ? '0.0.0.0' : '127.0.0.1';
+var host = process.env.HOST || '127.0.0.1';
```

Comment:
Why is a example specific to Heroku? The fewest npm servers are running on Heroku.
A example contained in an npm package should be either declared as such or be generic.

So ... you ask me where the HOST env is coming from - wehere is the PORT env coming from? I'd guess the same place, no? 🤔  The original example code was: "When there is a PORT env var set use `0.0.0.0`, otherwise use localhost - which doesn't make sense. It actually makes sense to have the bind HOST injected from outside - no?